Makefile

```diff
@@ -13,6 +13,8 @@ doctest-modules-local-namespace: export PYVISTA_OFF_SCREEN = True
 
 stylecheck: codespell pydocstyle lint
 
+format: isort stylize
```

Comment:
Now we can add format to CI, right?

Reply:
Likely we can add this in pre-commit `.pre-commit-config.yaml`. I think we should move from using the `Makefile` to pre-commit, but we can do that in a follow-up PR and get this merged as this is a huge PR.

Reply:
Yeah, we don't want the CI to run `format` but only `lint`/`stylecheck`. Having the CI perform the formatting is a recipe for merge conflicts and user headaches.

I'm not familiar with the pre-commit stuff. I think it'd be nice to get that in a separate PR where we can add installation/usage instructions as well